manifests/resource.pp

```diff
@@ -23,32 +23,34 @@
 #  [ha_primary] If the resource is being applied on the primary host.
 #  [initial_setup] If this run is associated with the initial setup. Allows a user
 #    to only perform dangerous setup on the initial run.
+#  [startup_parameters] Manage startup{} parameters (like wfc-timeout)
 define drbd::resource (
-  $host1           = undef,
-  $host2           = undef,
-  $ip1             = undef,
-  $ip2             = undef,
-  $res1            = undef,
-  $res2            = undef,
-  $cluster         = undef,
-  $secret          = false,
-  $port            = '7789',
-  $device          = '/dev/drbd0',
-  $mountpoint      = "/drbd/${name}",
-  $automount       = true,
-  $owner           = 'root',
-  $group           = 'root',
-  $protocol        = 'C',
-  $verify_alg      = 'crc32c',
-  $rate            = false,
-  $disk_parameters = false,
-  $net_parameters  = false,
-  $manage          = true,
-  $ha_primary      = false,
-  $initial_setup   = false,
-  $fs_type         = 'ext4',
-  $mkfs_opts       = '',
-  $disk            = undef,
+  $host1              = undef,
+  $host2              = undef,
+  $ip1                = undef,
+  $ip2                = undef,
+  $res1               = undef,
+  $res2               = undef,
+  $cluster            = undef,
+  $secret             = false,
+  $port               = '7789',
+  $device             = '/dev/drbd0',
+  $mountpoint         = "/drbd/${name}",
+  $automount          = true,
+  $owner              = 'root',
+  $group              = 'root',
+  $protocol           = 'C',
+  $verify_alg         = 'crc32c',
+  $rate               = false,
+  $disk_parameters    = false,
+  $net_parameters     = false,
+  $manage             = true,
+  $ha_primary         = false,
+  $initial_setup      = false,
+  $fs_type            = 'ext4',
+  $mkfs_opts          = '',
+  $disk               = undef,
+  $startup_parameters = undef,
```

Comment:
Could this be `Optional[Hash[String, String]] $startup_parameters = undef` instead?